src/pages/orphaned.js

```diff
@@ -0,0 +1,369 @@
+                              <Bookmark
+                                ruleId={rule.frontmatter.guid}
+                                className="category-bookmark"
+                              />
+                              <button className="tooltip">
+                                <a
+                                  target="_blank"
+                                  rel="noopener noreferrer"
+                                  href={`/rules/admin/#/collections/rule/entries/${rule.frontmatter.uri}/rule`}
+                                  className="tooltip tooltip-button"
+                                  onClick={() => {
+                                    appInsights.trackEvent({
+                                      name: 'EditMode-NetlifyCMS',
+                                    });
+                                  }}
+                                >
+                                  <FontAwesomeIcon
+                                    icon={faPencilAlt}
+                                    size="lg"
+                                    className="text-ssw-red bookmark-icon"
+                                  />
+                                </a>
+                                <span className="tooltiptext">Edit</span>
+                              </button>
+                              <button className="tooltip">
+                                <a
+                                  target="_blank"
+                                  rel="noopener noreferrer"
+                                  href={`https://github.com/SSWConsulting/SSW.Rules.Content/tree/${process.env.CONTENT_BRANCH}/rules/${rule.frontmatter.uri}/rule.md`}
+                                  className="tooltip tooltip-button"
+                                  onClick={() => {
+                                    appInsights.trackEvent({
+                                      name: 'EditMode-GitHub',
+                                    });
+                                  }}
+                                >
+                                  <FontAwesomeIcon
+                                    icon={faGithub}
+                                    size="lg"
+                                    className="text-ssw-red bookmark-icon"
+                                  />
+                                </a>
+                                <span className="tooltiptext">
+                                  Edit in GitHub
+                                </span>
+                              </button>
+                            </div>
+                          </div>
+                        </section>
+
+                        <section
+                          className={`rule-content mb-4
+                            ${selectedOption === 'all' ? 'visible' : 'hidden'}`}
+                        >
+                          <div
+                            dangerouslySetInnerHTML={{ __html: rule.html }}
```

Comment:
It's not ideal that we have to use `dangerouslySetInnerHTML` here, but this is already being used by `rule.js` to render the majority of rules.

Reply:
I have raised this concern before and we decided it was ok as the html being rendered comes from Markdown which has gone through the PR approval process.
This should ensure nothing malicious is rendered.

Hey @BrookJeynes 
1. Its probably a good idea to document this, most devs will have the same question.
2. Check the docs rule to find the best place for this.
3. Add a comment in the code with a link to docs wherever `dangerouslySetInnerHTML` is used